src/layouts/Layout.astro

```diff
@@ -27,7 +28,13 @@ const { title } = Astro.props;
   </head>
   <body class="bg-dark-blue">
     <Header />
-    <slot />
+    <div class="content-container">
+      <FilterSidebar />
```

Comment:
Migrate the `FilterSidebar` from the `Layout` to `index.astro` as we would only want filtering on the home page.